aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/SyncProcessor.java

```diff
@@ -99,18 +101,16 @@ public static Builder builder() {
      * @return An Rx {@link Completable} which can be used to perform the operation.
      */
     Completable hydrate() {
-        ModelClassComparator modelClassComparator =
```

Comment:
Can you delete the `ModelClassComparator` now?

Reply:
Yes! Done.